static/src/javascripts/projects/common/modules/identity/formstack-iframe-embed.js

```diff
@@ -0,0 +1,295 @@
+// @flow
+
+/**
+    Formstack - composer integration
+
+    This script runs INSIDE a formstack iframe and is initialised by the NGW
+    requireJs setup
```

Comment:
I guess this is no longer initialised by the NGW (Next Gen Web?) RequireJS setup 😄

Reply:
Hah, yeah I'll bring this comment up-to-date!